main.py

```diff
@@ -1,10 +1,69 @@
+import json
+
+from django.utils import timezone
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+
+        if not Race.objects.filter(
+                name=player_info["race"]["name"]
+        ).exists():
+            Race.objects.create(
+                name=player_info["race"]["name"],
+                description=player_info["race"]["description"],
+            )
```

Comment:
Use [this](https://docs.djangoproject.com/en/dev/ref/models/querysets/#get-or-create) method to avoid getting in the 61-63 lines